lago/prefix.py

```diff
@@ -493,7 +493,17 @@ def _handle_file_disk(self, disk_spec):
         if url:
             disk_spec['path'] = self._retrieve_disk_url(url, disk_path)
         else:
-            disk_spec['path'] = disk_path
+            # Create a copy of the file or use the original
+            if disk_spec.get('make_a_copy'):
+                LOGGER.info("Making a copy")
+                dest_path = self._generate_disk_path(
+                    os.path.basename(disk_spec['path'])
+                )
```

Comment:
not sure - would we have a problem here with >1 disks from the same image and ``'make_a_copy=true'``?